packages/workspace/src/generators/move/lib/move-project-configuration.spec.ts

```diff
@@ -24,7 +24,7 @@ describe('moveProjectConfiguration', () => {
       relativeToRootDestination: 'apps/subfolder/my-destination',
     };
 
-    tree = createTreeWithEmptyWorkspace();
+    tree = createTreeWithEmptyWorkspace(version);
```

Comment:
Remark related to this, would it be preferable to set the default version of `createTreeWithEmptyWorkspace(version = 1)` to `2` instead of `1` since v2 format is the most used in the repo?

Reply:
Yeah, we need to make that change but not as part of this PR. It's a breaking change since this utility is exposed as part of the devkit and folks could be relying on the v1 format. We'll discuss this and make the change in a separate PR ensuring it's included in the release notes as a breaking change.